introduction.md

```diff
@@ -1,3 +1,7 @@
 Hi I am Larry
 
 Javascript
+
+*Javascript
+*java
+*node.js
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?